test-suite/testkit/tests/runtime_api/proto/test_service.proto

```diff
@@ -0,0 +1,19 @@
+// Copyright 2019 The Exonum Team
```

Comment:
I suggest to remove the copy-right note and make this file really small, so that we can check not only name, but also the file contents returned from the Rust runtime API.